src/app/components/SearchResults/SearchHeader.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
It doesn't seem like you are using this file. Also, I don't think this is the best name and we try not to create layouts using `table`. So it'd be prefer if you do it with `div`s and style it rather than use tables. They should only be used for tabular data.